2/contrib/s2i/run

```diff
@@ -466,6 +466,7 @@ if [[ $# -lt 1 ]] || [[ "$1" == "--"* ]]; then
   set -x
   exec java $JENKINS_JAVA_OPTIONS -Duser.home=${HOME} \
             -Djavamelody.application-name=${JENKINS_SERVICE_NAME} \
+            -Djavamelody.displayed-counters=log,error \ # a discover was made upstream that if the monitor plugin is installed, it creates httpsession's via its filter, which impact the login plugin bearer token support, this setting turns that off
```

Comment:
how about setting it in JAVA_OPTIONS so users can at least override the entire JAVA_OPTIONS to change the behavior? above